contrib/inventory/vmware.py

```diff
@@ -185,7 +185,7 @@ def _get_obj_info(self, obj, depth=99, seen=None):
             if depth <= 0:
                 return obj_unicode
             d = {}
-            for attr in dir(obj):
+            for attr in obj.__dict__:
```

Comment:
I'm a little bit in doubt about these. `dir` returns more items than `__dict__`.